data/json/items/gun/12mm.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "id": "hk_g80",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "light_gray",
+    "name": "H&K G80 Railgun",
```

Comment:
Every other file in that directory has specific ordering of fields. It would be a shame for a handful of new entries not to match this considering the prior work involved in refactoring the JSON to achieve this.